presto-docs/src/main/sphinx/release/release-0.153.rst

```diff
@@ -6,10 +6,72 @@ General Changes
 ---------------
 
 * Fix incorrect results for grouping sets when ``task.concurrency`` is greater than one.
+* Fix silent numeric overflow when casting ``INTEGER`` to large ``DECIMAL``
+  types.
```

Comment:
Don't wrap single words. This makes it harder to read. (wrapping is for readability)